src/frontend/komponenter/Fagsak/Vedtak/FeilutbetaltValuta/FeilutbetaltValuta.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { useEffect, useState } from 'react';
+
+import styled from 'styled-components';
+
+import { AddCircle } from '@navikt/ds-icons';
+import { Button, Heading, Table } from '@navikt/ds-react';
+import { CopyToClipboard } from '@navikt/ds-react-internal';
+import { NavdsSemanticColorLink } from '@navikt/ds-tokens/dist/tokens';
+
+import type { IRestFeilutbetaltValuta } from '../../../../typer/eøs-feilutbetalt-valuta';
+import { periodeToString } from '../../../../utils/kalender';
+import FeilutbetaltValutaPeriode from './FeilutbetaltValutaPeriode';
+import NyFeilutbetaltValutaPeriode from './NyFeilutbetaltValutaPeriode';
+
+interface IFeilutbetaltValuta {
+    behandlingId: number;
+    fagsakId: string | undefined;
+    feilutbetaltValutaListe: IRestFeilutbetaltValuta[];
+    settErUlagretNyFeilutbetaltValutaPeriode: (erUlagretNyFeilutbetaltValuta: boolean) => void;
+    erLesevisning: boolean;
+    skjulFeilutbetaltValuta: () => void;
+}
+
+const FlexColumnDiv = styled.div`
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+    margin-bottom: 3.5rem;
+    margin-top: 2rem;
+`;
+
+const FlexRowDiv = styled.div`
+    display: flex;
+    justify-content: space-between;
+`;
+
+const KopierTilNøsKnapp = styled(CopyToClipboard)`
+    :not(:hover):not(:active) {
+        color: ${NavdsSemanticColorLink};
+    }
+`;
+
+const FeilutbetaltValuta: React.FC<IFeilutbetaltValuta> = ({
+    feilutbetaltValutaListe,
+    settErUlagretNyFeilutbetaltValutaPeriode,
+    erLesevisning,
+    skjulFeilutbetaltValuta,
+    behandlingId,
+    fagsakId,
+}) => {
+    const [ønskerÅLeggeTilNyPeriode, settØnskerÅLeggeTilNyPeriode] = useState(
+        feilutbetaltValutaListe.length === 0
+    );
+
+    useEffect(() => {
+        settErUlagretNyFeilutbetaltValutaPeriode(ønskerÅLeggeTilNyPeriode);
+    }, [ønskerÅLeggeTilNyPeriode]);
```

Comment:
Må dette være i en useEffect? Trodde React var smart nok til å ikke kalle sett-funksjoner dersom de blir kalt med samme verdi. I så fall kan den jo bare være den del av det som skjer ved render:
```suggestion
    settErUlagretNyFeilutbetaltValutaPeriode(ønskerÅLeggeTilNyPeriode);
```

Reply:
Eventuelt kunne man bundla sammen dette. Typ:
```
    const oppdaterØnskerÅLeggeTilNyPeriode(ønskerÅLeggeTilNyPeriode: boolean) => {
        settØnskerÅLeggeTilNyPeriode(ønskerÅLeggeTilNyPeriode)
        settErUlagretNyFeilutbetaltValutaPeriode(ønskerÅLeggeTilNyPeriode)
    }
```

Så er det lettere å se konsekvensene av koden man kjører

Reply:
https://beta.reactjs.org/learn/you-might-not-need-an-effect#adjusting-some-state-when-a-prop-changes

Reply:
https://beta.reactjs.org/learn/you-might-not-need-an-effect#adjusting-some-state-when-a-prop-changes